core/src/executor/evaluate/function.rs

```diff
@@ -1,3 +1,5 @@
+use super::ChronoFormatError;
```

Comment:
Combine it with the import statement below

Reply:
Oh, thanks for poting this out, I will apply this ASAP